schema.ts

```diff
@@ -143,11 +156,6 @@ export enum ValueHintType {
    * ```
    */
   Reference = 'reference',
-  /**
-   * Indicates to interpret and render a value as a file attachment. The provided value should be a URL
-   * pointing to a file of a Coda-supported type. Coda will ingest the file and host it from Coda infrastructure.
-   */
-  Attachment = 'attachment',
```

Comment:
hmm this will be a breaking change i believe. is that intentional? maybe relates to whether we need FileReference and if not we should stick to `Attachment`

Reply:
Ah yup this would be breaking, I should have called this out for discussion. If we do want `FileReference` it seemed confusing to me that we had something that was just called `Attachment`, so figured it would be good to rename this in conjunction.

Reply:
Yeah, I wouldn't rename this. `Attachment` already means `FileAttachment` and we already have support for this as a return value, so I wouldn't want to touch it.